### PR TITLE
Update TACC-Stampede2.yaml

### PR DESCRIPTION
**Pull request by tanyalevshina**
added WLCGInformation block (ApelNormalFactor) for TACC-Stampede2

### CODE REVIEW

**Comment by djw8605** [comment]
Brian, could you confirm that the WLCG is at the right indention level. It doesn’t look right at first glance. 

-Derek

File: topology/University of Texas at Austin/Texas Advanced Computing Center/TACC-Stampede2.yaml [hunk 1]
```diff
@@ -35,6 +35,8 @@ Resources:
     VOOwnership:
       CMS: 50
       ATLAS: 50
+  WLCGInformation:
+      APELNormalFactor: 18
```
Comment (matyasselmeci) [inline topology/University of Texas at Austin/Texas Advanced Computing Center/TACC-Stampede2.yaml:39]:
```suggestion
    WLCGInformation:
        APELNormalFactor: 18
```

**Comment by matyasselmeci** [comment]
Good catch!

**Comment by matyasselmeci** [comment]
Looks like the schema won't validate unless you specify a few more WLCGInformation attributes:
- InteropMonitoring
- InteropAccounting
- KSI2KMin
- KSI2KMax

(the first two are bools and the last two are floats; the other WLCGInformation attributes are strings so when topology fills them in with an empty string, that fits the schema. This should probably be fixed in topology.)

**Review by matyasselmeci — APPROVED** [review]

**Pull request merged**